tests/test_baselines.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+
+"""Test non-parametric baseline models."""
+
+from typing import Any, MutableMapping
+
+import torch
+import unittest_templates
+
+from pykeen.datasets import Nations
+from pykeen.models import MarginalDistributionBaseline
+
+
+class MarginalDistributionBaselineTests(unittest_templates.GenericTestCase[MarginalDistributionBaseline]):
+    """Tests for MarginalDistributionBaseline."""
+
+    #: The batch size
+    batch_size: int = 3
+
+    #: The tested class
+    cls = MarginalDistributionBaseline
+
+    def _pre_instantiation_hook(self, kwargs: MutableMapping[str, Any]) -> MutableMapping[str, Any]:  # noqa: D102
+        kwargs = super()._pre_instantiation_hook(kwargs=kwargs)
+        dataset = Nations()
+        self.factory = dataset.training
+        kwargs["triples_factory"] = self.factory
+        return kwargs
+
+    def test_score_t(self):
```

Comment:
@mberr add tests for all combinations of entity_margin=False/True and relation_margin=False/True

Reply:
15f91a8